blog/2024-02-21/index.md

```diff
@@ -0,0 +1,28 @@
+---
+slug: improved-build-validation
+title: Improved build validation and switch to libappstream
+authors: [barthalion]
+tags: [flathub, moderation, linter]
+---
+
+Flathub's automatic build validation is more thorough now, and includes checks for issues we would have previously flagged only manually. There is a chance that if your app has been passing the continuous integration checks previously, it will fail now.
```

Comment:
A little more helpful as a standalone snippet with a positive ending to encourage people to read on.

```suggestion
Flathub's automatic build validation is more thorough now, and includes checks for issues we previously would have only flagged manually. There is a chance that if your app has been passing the continuous integration checks previously, it will fail now; here's why, and what to do about it.
```